core/src/ldml/C7532_ldml_updating.md

```diff
@@ -0,0 +1,76 @@
+- `core/src/kmx/kmx_plus.cpp`
+    - add validate implementation for the section and any new structs
+    - update `kmx_plus::kmx_plus()` to include the loader
+- `common/web/types/src/kmx/kmx-plus.ts`
+    - Also update class KMXPlusFile to include the actual binary format (coordinate with `kmx_plus.h`)
+- E/C `common/web/types/src/ldml-keyboard/ldml-keyboard-xml-reader.ts` if there’s any changing to the boxing needed
+
+## In-memory data: Phase 2
+
+- first the compiler tests
+    - add a section in `developer/src/kmc-keyboard/test/fixtures/sections` if needed
+    - add a test case such as `developer/src/kmc-keyboard/test/test-key2.ts`
+- add a compiler
+    - `developer/src/kmc-keyboard/compiler/key2.ts`
+    - link it in to `developer/src/kmc-keyboard/src/compiler/compiler.ts`
+        - add the import
+        - add to `SECTION_COMPILERS`
+
+    - Note: the in-memory compiler can affect `basic.kmx` even _before_ you add the section writing code. How? Simple… `Strs.allocString()` is called for the in-memory structures, so the string table will start growing even before those strings are actually used by the new sections.  This is why it's fine to ignore the basic failure until you actually do the KMXPlus write.
+
+## Writing out
+
+- The moment you've been waiting for! Crack open `common/web/types/src/kmx/kmx-plus-builder/kmx-plus-builder.ts` and do it.
+    - Add `import { BUILDER_DISP, build_disp } from './build-disp.js';` to the top (and a new file to go with it) — and, in order
+    - Add `private sect_disp: BUILDER_DISP` to `class  KMXPlusBuilder {` — and, in order
+    - add `this.sect_disp = build_disp(this.file.kmxplus, this.sect_strs);` to the `build()` function.  Include any other sections that need to be cross referenced.
+    - Update `finalize_sect` and add `offset = this.finalize_sect_item(this.sect_disp, offset);`
+    - Finally, add `this.emitSection(file, this.file.COMP_PLUS_DISP, this.sect_disp);` to `compile()` — and, in order.
+        - Note that some variable length parts (such as the actual text data in `strs`) are sometimes in a separate emit function. Anything that's not in the `COMP_PLUS_STRS` `r.Struct` definition needs one of these.
+        - Also note that restructure will happily ignore (write zeros for) any fields where the BUILDER_* fields don't match the COMP_PLUS_* fields.
+
+- update basic.xml and basic.txt
+    - Tweak `eveloper/src/kmc-keyboard/test/fixtures/basic.xml` as needed
+    - You can use `developer/src/kmc-keyboard/build.sh build-fixtures` which will generate these. The two .kmx files are supposed to match: if not, fix `basic.txt` or fix other bugs.
+        - `developer/src/kmc-keyboard/build/test/fixtures/basic-txt.kmx` - KMX generated from basic.txt.
+        - `developer/src/kmc-keyboard/build/test/fixtures/basic-xml.kmx` - KMX generated from basic.xml.
+        - `developer/src/kmc-keyboard/build/test/fixtures/basic-xml.kvk` - KVK generated from basic.xml.
+
+## more to come
```

Comment:
novelization, stage and direct-to-tv rights are still available for this gripping saga